packages/dbml-parse/src/lib/analyzer/validator/elementValidators/table.ts

```diff
@@ -134,7 +134,7 @@ export default class TableValidator implements ElementValidator {
 
     const maybeNameFragments = destructureComplexVariable(name);
     if (maybeNameFragments.isOk()) {
-      const nameFragments = maybeNameFragments.unwrap();
```

Comment:
why change this em? are we going to mutate the name fragements?

Reply:
yeah, `nameFragment` is mutated and `maybeNameFragments` is reused below so I cloned it.